frontend/sw360-portlet/src/main/resources/META-INF/resources/html/homepage/myprojects/view.jsp

```diff
@@ -66,7 +66,25 @@
                     <core_rt:if test="${userRoles==null||userRoles.SECURITY_RESPONSIBLES}">checked="checked"</core_rt:if>>
                     <label class="form-check-label" for="securityResponsible">Security Responsible</label>
                 </li>
+                <li><hr class="my-2" /></li>               
+                <li class="dropdown-header">Clearing State</li>
```

Comment:
It would be good to read the labels from i18N properties.

Reply:
Followed the existing approach,which is working in production for roles label.

Reply:
hm, it was for internationalization. But it's ok for now.